packages/reducers/src/makeEnhancer.js

```diff
@@ -8,6 +23,29 @@ import composeReducers from './composeReducers';
 
 export const storeInterface = makeStoreInterface('reducers');
 
+const cleanUpReducer = (state, action) => {
+	if (action.type !== '@redux-tools/REDUCERS_EJECTED') return state;
+
+	const namespaceGroup = action.meta.feature || DEFAULT_FEATURE;
+
+	const pathToNamespace = [namespaceGroup, action.meta.namespace];
+	const pathToReducer = when(always(action.meta.namespace), concat(pathToNamespace))([]);
+
+	const cleanReducers = reduce((state, reducerName) =>
```

Comment:
whew, uhh.. pointfree is not always the way to go, I think that doing sth like

```
const removeEjectedState = prevState => reduce((nextState, reducerKey) => dissoc(), prevState, action.payload))
```

would be more readable (along with the renames)